x-pack/plugins/endpoint/public/embeddables/resolver/models/process_event_test_helpers.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { ProcessEvent } from '../types';
+type DeepPartial<T> = { [K in keyof T]?: DeepPartial<T[K]> };
+
+export function mockProcessEvent(
+  parts: {
+    data_buffer: { node_id: ProcessEvent['data_buffer']['node_id'] };
```

Comment:
See comments about what a "process event" is -- none of these fields exist in process events that we ship today to Elasticsearch or in the events we'll ship in the future.